hashicorp/waypoint
# ui: updated the ui of list items on deployments tab

## Notes
- The URL shown on each list item is the **_deployment URL_**, the designs state it should be the _**release URL**_, but in consideration of the context and after discussion, we chose to show the deployment URL here
- Currently the frontend does not receive the commit message with git commit information at all, so that needs to be implemented later
- As noted in some other PRs, currently there are no animated icons in `flight`, so the `Deploying...` icon is the `clock` icon until the animated loading one is available

## Screenshots
**_Note the left sidebar_**
### Before
<img width="1438" alt="Screen Shot 2022-01-06 at 3 06 24 PM" src="https://user-images.githubusercontent.com/60155296/148444496-12915d91-0696-47e7-943d-a0b7a7fb7313.png">

### After
<img width="1440" alt="Screen Shot 2022-01-12 at 12 33 14 PM" src="https://user-images.githubusercontent.com/60155296/149191901-15fb4e3f-1e89-4163-ac33-586e3c1dc8b5.png">


## How to test
1. Pull down the branch `git checkout ui/deployment-card-state`
2. `yarn start`
3. Check the deployments tab of an application
4. Optional: test UI against a real Waypoint server
